db-client/src/db.ts

```diff
@@ -1,4 +1,3 @@
-import deepFreeze, { DeepReadonly } from 'deep-freeze';
```

Comment:
Note that this PR includes parts of #182.

Reply:
Thanks!
Forgot that I started from that branch..